packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -39,11 +39,10 @@ export const PlayerHero: React.FC<Props> = ({ player }) => {
   return (
     <ProfileSection>
       <VStack spacing={8}>
-        <Avatar
+        <PlayerAvatar
           w={{ base: 32, md: 56 }}
           h={{ base: 32, md: 56 }}
-          src={getPlayerImage(player)}
-          name={getPlayerName(player)}
+          {...{ player }}
```

Comment:
you could just do
```suggestion
          player={player}
```
right?

Reply:
I think it works but, who wants to see player twice!? :) 

@dysbulic showed me this lil trick and i forgot about it.